### DISCUSSION TITLE
Problem logging into the client after upgrade

### DISCUSSION THREAD

**Discussion by kevinreyn** [jlesage/docker-crashplan-pro#469] [General]
Curious if there is a known issues with client access on synology headless implementation after latest release. 

I made sure docker is latest version avail from package center. 
Pulled latest cpp image 
reset and restarted the container. 
Attempted to access the client via ip:5800 get prompted for UID PAssword and Authenticator key 
When I enter the correct values i get the following 

unable to fetch crashplan server environment insufficient randomness

Reviewing the logs I see the following - though it looks like the services all start up 
error: Ignoring /opt/base/etc/logrotate.conf because it is writable by group or others.
ERROR: service 'CrashPlanEngine' failed to be started: not ready after 30000 msec, giving up.

Any thoughts on this would be appreciated

**Comment by jlesage** [comment]
What CrashPlan server is used ? You should have an URL displayed behind the window that asks for the password. It should start with something like `https://console.us2.crashplan.com`.

**Reply by kevinreyn** [reply]
Yes that is in the dialog that is presented I think - I cannot interact with it and full url is not visible but looks like that 
If I navigate to that address I can log in just fine.

**Reply by kevinreyn** [reply]
for additional color 
Prior to the upgrade I would simply hit the ip of my Nas x.x.x.x:5800 and it would present me with the client login etc. I would log in and then be gtg.  Its at that point right now that I get the dialog then the subsequent error

**Reply by jlesage** [reply]
So what is the address of the server in that URL?

The latest version of CrashPlan changed the way login is performed. Note that in any case, you can copy the presented URL, paste it in your browser and perform the login from there.

**Reply by kevinreyn** [reply]
maybe thats where I am confused. 
I used to connect to my nas at my local address. 192.168.x.x:5800 and it was the front end to the service running on my NAS 
Do we now just use the front end at the address https://console.us2.crashplan.com to manage our instances?  

also lookin at the logs after it has been up for quite some time the last entry in the log is "All services started" but by now I would have seen some logging of synching backup and then transferring so wondering if I have something boogered

**Reply by kevinreyn** [reply]
btw thank you for maintaining this - I would have bailed on crash plan years ago had I not discovered your conversion

**Reply by jlesage** [reply]
Connecting to `192.168.x.x:5800` is still the way to go. But you can also manage your instance from crashplan.com (there is nothing new there, this has always been possible).

**Reply by jlesage** [reply]
To come back to your `unable to fetch crashplan server environment insufficient randomness` error: this might be a temporary problem. Do you still have the same error when trying to login ?

**Comment by kevinreyn** [comment]
Only when attempting to hit the local client on 192.168.x.x:5800
I can log in just fine through the web site

**Reply by jlesage** [reply]
Do you get the same error if you execute `docker exec <container name> curl https://console.us2.crashplan.com` ?

Also, could you share the output of the following commands:
- `docker exec <container name> ls -l /dev/random`
- `docker exec <container name> ls -l /dev/urandom`

**Comment by nckwltn** [comment]
@kevinreyn 

I also had the same issue with the "unable to fetch" error. Running the container on my synology nas.

I was able to get logged in by opening the side window on the left of the screen. The one that auto hides. From there I could copy the provided URL to my local clipboard and auth outside of the virtual display inside the browser.  Copying from the "copy" button next to the URL did not bring the text to my local clipboard.

Not sure if I will have to go through this procedure each time, but I login to crashplan infrequently, so not a HUGE issue for me


@jlesage THANK YOU for updating to the latest version!!!

**Reply by jlesage** [reply]
See my previous comment and https://github.com/jlesage/docker-crashplan-pro/issues/470 if you want to help debugging/fixing this.

**Comment by kevinreyn** [comment]
Sorry for the delay

the output you asked for

ash-4.4# docker exec crashplan-pro curl https://console.us2.crashplan.com
  % Total    % Received % Xferd  Average Speed   Time    Time     Time
 Current
                                 Dload  Upload   Total   Spent    Left
 Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--
  0
ash-4.4#

ash-4.4# docker exec crashplan-pro ls -l /dev/random
crw-rw-rw-    1 root     root        1,   8 Aug  4 23:11 /dev/random
ash-4.4# docker exec crashplan-pro ls -l /dev/urandom
crw-rw-rw-    1 root     root        1,   9 Aug  4 23:11 /dev/urandom
